cpp/src/gandiva/proto/Types.proto

```diff
@@ -219,8 +219,10 @@ message InNode {
   optional TreeNode node = 1;
   optional IntConstants intValues = 2;
   optional LongConstants longValues = 3;
-  optional StringConstants stringValues = 4;
-  optional BinaryConstants binaryValues = 5;
+  optional FloatConstants floatValues = 4;
```

Comment:
You should not likely change the existing field numbers to not break backward compatibility

Reply:
done

Reply:
> > > @projjal, can you please help me discover why the test TestInFloat is failing in this travis build: https://travis-ci.com/github/apache/arrow/jobs/494494759 ? I already tried to explicitly declare the float values, but it didn't seen to make any difference.
> > 
> > 
> > Might be some round-off noise in floating point addition in your test. How about test with a single floating point field instead of sum of two fp fields and see if it passes.
> 
> Changed to only one field, but still not passing, do you know any other way I can use to avoid this round-off noise?

Round-off noise was a guess. Since its even not working with direct field, its probably not the issue.